bach/sql_models/util.py

```diff
@@ -70,21 +71,30 @@ def quote_identifier(dialect: Dialect, name: str) -> str:
     raise DatabaseNotSupportedException(dialect)
 
 
-def quote_string(value: str) -> str:
+def quote_string(dialect_engine: Union[Dialect, Engine], value: str) -> str:
     """
     Add single quotes around the value and escape any quotes in the value.
 
     This is in accordance with the Postgres string notation format, no guarantees for other databses.
     See https://www.postgresql.org/docs/14/sql-syntax-lexical.html#SQL-SYNTAX-CONSTANTS
```

Comment:
I know I forgot to update the description above. But I think we need to have a more database independent description here.


Perhaps something as simple as this?
```suggestion
    Quote and escape string value for the given database dialect.
```

and then move the link to postgresql documentation below in the code as an inline comment?

Reply:
then we should do it for ```quote_identifier```, I just followed the same pattern

Reply:
Fair point, I forgot to update  `quote_identifier()`'s docstring after making it multi-database. It still talked about Postgres being a default etc. Created a small PR to fix that: https://github.com/objectiv/objectiv-analytics/pull/642

Reply:
P.S. I didn't mean that we should remove the example. We should just update the description to cover both the Postgres and BigQuery implementation, whereas previously it only described the Postgres part of the implementation.